docs/components/navbars.html

```diff
@@ -99,7 +99,7 @@ <h1>Navbars</h1>
             <li><a href="#">Item 3</a></li>
             <li><a href="#">Item 4</a></li>
             <input class="inp-text"></input>
-            <input type="checkbox" class="hm"></input>
+	    <span class="hm"></span>
```

Comment:
Please run a formatter before submitting any changes requested above. Automation using GitHub Actions will not be used on Webments repositories and it should be done locally before committing & pushing to remote.

Reply:
Its just a one more space. Can be solved easily.

Reply:
That wasn't a warning specifically for line 102, it was in general.

Reply:
The thing above is recently a problem of Github.

Reply:
ok